agent/util-scripts/pbench-tool-data-sink

```diff
@@ -0,0 +1,551 @@
+#!/usr/bin/env python3
```

Comment:
Alot of this should be moved into lib/pbench/agent, lets keep the pyhton bits in the pbench namespace

Reply:
Agreed.

It just seemed that with so much change, it was not worth adding more change to this PR.